src/main/java/de/siegmar/logbackgelf/GelfEncoder.java

```diff
@@ -410,13 +413,13 @@ public byte[] encode(final ILoggingEvent event) {
 
         final byte[] json = gelfMessageToJson(gelfMessage);
 
-        if (appendNewline) {
-            final byte[] sep = System.lineSeparator().getBytes(StandardCharsets.UTF_8);
-            final ByteBuffer bb = ByteBuffer.allocate(json.length + sep.length);
+        if (appendNewline){
+            final byte sep = '\n';//System.lineSeparator().getBytes(StandardCharsets.UTF_8);
+            final ByteBuffer bb = ByteBuffer.allocate(json.length + 1);// sep.length
```

Comment:
I assume this breaks windows console logging – the feature this was built for...

Reply:
Windows Console supports both \r\n and \n
Please try yourself.
Not only System.out.print("line1\nline2\r\nline3") works, but also $ type FileWithLfEols.java

Reply:
Still don't see a reason to change this. Windows line ending is \r\n even if the terminal now supports unix line endings.